nevergrad/parametrization/container.py

```diff
@@ -50,6 +50,18 @@ def value(self, value: tp.Tuple[tp.Any]) -> None:
             raise ValueError(f"{cls} value must be a tuple of size {len(self)}, got: {value}.\nCurrent value: {self.value}")
         for k, val in enumerate(value):
             core.as_parameter(self[k]).value = val
+    
+    def get_ranges(self) -> tp.List[tp.List[int]]:
```

Comment:
please move this to parametrization/helpers.py which is well suited for this type of functions. I'm pretty sure it will change signature (because one day you'll need the type as well), so this is unstable and should preferably not be included in the public API.

Also, concerning typing `tp.List[tp.Tuple[int, int]]` will more accurate